core/kazoo_services/src/kz_services_plans.erl

```diff
@@ -306,7 +317,7 @@ merge_override(ServicesJObj, Overrides) ->
 %%------------------------------------------------------------------------------
 -type mergable() :: plans() | plans_list().
 -spec merge(mergable()) -> kz_services_plan:plan().
-merge([Head|Tail]=Plans) when is_list(Plans) ->
+merge([Head|Tail]=Plans) ->
```

Comment:
what if the list is empty? Also I think its better to check if it is list by `is_list`.

Reply:
@icehess trying to enforce never merging no plans, would result in the requester doing work it doesn't need to.  Also, why the keep the guard?  Its guaranteed by the function clause...